packages/maps/src/components/ChatbotIframe.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import ChatbotIcon from "../assets/icons/chatbot.png";
+import { sendDataToChatbot } from "utils/chatbot";
+
+const ChatbotIframe = ({ selectedData, degrees }: { selectedData: any; degrees: any }) => {
+  const [isChatOpen, setIsChatOpen] = useState(false);
+
+  const toggleChat = () => {
+    setIsChatOpen(!isChatOpen);
+  };
+
+  const chatbotUrl = "http://localhost:3000";
```

Comment:
make it an env variable

Reply:
[commit](https://github.com/Probable-Futures/apps-public/pull/1/commits/0182cd894c4aa3bcdd852408ce9d0e7e33b97ce7)